testsuite/features/support/env.rb

```diff
@@ -131,21 +130,17 @@ def capybara_register_driver
   log "This scenario took: #{current_epoch - @scenario_start_time} seconds"
   if scenario.failed?
     begin
-      Timeout.timeout(DEFAULT_TIMEOUT) do
```

Comment:
This is not affecting only BV, but any kind of test suite.
Could you please rephrase the PR title and description to notice that fact?